force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -11813,6 +11837,54 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>pmtPaidPaymentDateRequired</shortDescription>
         <value>A paid Payment must have a Payment Date.</value>
     </labels>
+    <labels>
+        <fullName>pmtRefundPaymentErrorMessage</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>pmtPaidPaymentDateRequired</shortDescription>
+        <value>There was an error processing the refund.</value>
+    </labels>
+    <labels>
+        <fullName>pmtPaymentNotFund</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Cannot find the payment in Elevate</shortDescription>
+        <value>Can not find the corressponding payment record in Elevate.</value>
+    </labels>
+    <labels>
+        <fullName>pmtPaymentNotRefundable</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Cannot refund the payment</shortDescription>
+        <value>Cannot refund a transaction twice or refund a refund.</value>
+    </labels>
+    <labels>
+        <fullName>pmtRefundPaymentMessage</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Refund Information</shortDescription>
+        <value>This will refund the payment made on {{VALUE}} in the amount of {{VALUE}}</value>
+    </labels>
+    <labels>
+        <fullName>pmtRefundProcessing</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Payment refund submitted for processing</shortDescription>
+        <value>Payment refund submitted for processing.</value>
+    </labels>
+    <labels>
+        <fullName>pmtRefundPaymentTitle</fullName>
+        <categories>Payment, Refund</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Refund Payment Title</shortDescription>
+        <value>Refund Payment</value>
```

Comment:
```suggestion
        <value>Refund Payment?</value>
```

Reply:
This label is used on both the Title and the confirm button. We can change the button to other label if needed.
<img width="849" alt="Screen Shot 2022-05-20 at 1 49 33 PM" src="https://user-images.githubusercontent.com/57271120/169608694-205d261d-f50d-4a8b-9913-89c7339e231e.png">

Reply:
I will create a separate label for the `?` one